source/cloud/gcp/vertex-ai.md

```diff
@@ -24,9 +24,10 @@ If you want to select a different GPU or select other hardware options you can s
 Once the instance has started select **OPEN JUPYTER LAB** and at the top of a notebook install the RAPIDS libraries you wish to use.
 
 ```bash
-!pip install cudf-cu11 dask-cudf-cu11 --extra-index-url=https://pypi.nvidia.com
-!pip install cuml-cu11 --extra-index-url=https://pypi.nvidia.com
-!pip install cugraph-cu11 --extra-index-url=https://pypi.nvidia.com
+pip install \
+    --extra-index-url={{rapids_pip_index}} \
+    "cudf-cu12>={{rapids_pip_version}}" "cuml-cu12>={{rapids_pip_version}}" \
+    "dask-cuda>={{rapids_pip_version}}"
```

Comment:
```suggestion
pip install \
    --extra-index-url={{rapids_pip_index}} \
    "cudf-cu12>={{rapids_version}}" "cuml-cu12>={{rapids_version}}" \
    "dask-cuda>={{rapids_version}}"
```

`rapids_pip_version` isn't defined in https://github.com/rapidsai/deployment/blob/main/source/conf.py ... I think we want `rapids_version` (which is the `YY.MM` version like `"24.10"`).

Reply:
Oh I see now, you're adding `rapids_pip_version` in #482.

Ok, approving this conditional on that one being merged first.